doc/concepts/configuration/configuration_code.rst

```diff
@@ -0,0 +1,341 @@
+        memtx_memory        = 33554432,
+        pid_file            = "tarantool.pid",
+        wal_max_size        = 2500
+    }
+    print('Starting ', arg[1])
+
+and suppose the environment variable LISTEN_URI contains 3301,
+and suppose the command line is ``~/tarantool/src/tarantool script.lua ARG``.
+Then the screen might look like this:
+
+..  code-block:: console
+
+    $ export LISTEN_URI=3301
+    $ ~/tarantool/src/tarantool script.lua ARG
+    ... main/101/script.lua C> Tarantool 2.8.3-0-g01023dbc2
+    ... main/101/script.lua C> log level 5
+    ... main/101/script.lua I> mapping 33554432 bytes for memtx tuple arena...
+    ... main/101/script.lua I> recovery start
+    ... main/101/script.lua I> recovering from './00000000000000000000.snap'
+    ... main/101/script.lua I> set 'listen' configuration option to "3301"
+    ... main/102/leave_local_hot_standby I> ready to accept requests
+    Starting  ARG
+    ... main C> entering the event loop
+
+If you wish to start an interactive session on the same terminal after
+initialization is complete, you can use :ref:`console.start() <console-start>`.
+
+
+..  _box-cfg-params-env:
+
+Environment variables
+---------------------
+
+Starting from version :doc:`2.8.1 </release/2.8.1>`, you can specify configuration parameters via special environment variables.
+The name of a variable should have the following pattern: ``TT_<NAME>``,
+where ``<NAME>`` is the uppercase name of the corresponding :ref:`box.cfg parameter <box-cfg-params-ref>`.
+
+For example:
+
+* ``TT_LISTEN`` -- corresponds to the ``box.cfg.listen`` option.
+* ``TT_MEMTX_DIR`` -- corresponds to the ``box.cfg.memtx_dir`` option.
+
+In case of an array value, separate the array elements by comma without space:
+
+..  code-block:: console
+
+    export TT_REPLICATION="localhost:3301,localhost:3302"
+
+If you need to pass :ref:`additional parameters for URI <index-uri-several-params>`, use the ``?`` and ``&`` delimiters:
+
+..  code-block:: console
+
+    export TT_LISTEN="localhost:3301?param1=value1&param2=value2"
+
+An empty variable (``TT_LISTEN=``) has the same effect as an unset one meaning that the corresponding configuration parameter won't be set when calling ``box.cfg{}``.
+
+
+
+..  _index-local_hot_standby:
+..  _index-replication_port:
+..  _index-slab_alloc_arena:
+..  _index-replication_source:
+..  _index-snap_dir:
+..  _index-wal_dir:
+..  _index-wal_mode:
+..  _index-checkpoint daemon:
+
+..  _box_cfg_params:
+
+
+Configuration parameters
+------------------------
+
+Configuration parameters have the form:
+
+:extsamp:`{**{box.cfg}**}{[{*{key = value}*} [, {*{key = value ...}*}]]}`
+
+Since ``box.cfg`` may contain many configuration parameters and since some of the
+parameters (such as directory addresses) are semi-permanent, it's best to keep
+``box.cfg`` in a Lua file. Typically this Lua file is the initialization file
+which is specified on the Tarantool command line.
```

Comment:
> it's best to keep ``box.cfg`` in a Lua file

It's simplest, but why the best? Cartridge, for example, allow to store box.cfg parameters in the cluster wide config.

And it is unclear how it is related to non-dynamic options.